meson.build

```diff
@@ -3741,7 +3746,7 @@ status = [
         'debug shell:                       @0@ @ @1@'.format(get_option('debug-shell'),
                                                               get_option('debug-tty')),
         'TTY GID:                           @0@'.format(tty_gid),
-        'users GID:                         @0@'.format(substs.get('USERS_GID')),
+        'users GID:                         @0@'.format(conf.get('USERS_GID')),
```

Comment:
hmm? can you explain?

Reply:
It's as @bluca wrote: we had two big `configuration_data` objects in the meson config. (There are in fact more, because I add one in this series, and there's one for efi… But those other have a handful over variables for specific purposes and don't matter). `conf` and `substs` were inherited from the original autotools system. In the past there was even a third set, but @yuwata removed it in 348b44372f36010d48d9a7dda14ef67155753a71. And the the two/three systems have very similar data, but with different variable names, because of historical reasons. This means we have duplicated work when changing things, and we need search for different variable names, etc.